misc/assets/obligee-action-wizard.svg

```diff
@@ -0,0 +1,542 @@
+    <text x="105" y="-5">yes</text>
+  </g>
+
+  <g transform="translate(1400 800)">
+    <use href="#result" />
+    <text class="caption">Affirmation</text>
+    <path class="arrow" d="M 100,0 h 100 v 200" />
+  </g>
+
+  <g transform="translate(800 900)">
+    <use href="#jump" />
+    <text class="caption">H*</text>
+  </g>
+
+  <g transform="translate(500 1000)">
+    <use href="#input" />
+    <text class="caption">Was it returned?</text>
+    <path class="arrow" d="M 0,20 v 180 h 200" />
+    <text x="5" y="35">no</text>
+    <path class="arrow" d="M 100,0 h 100" />
+    <text x="105" y="-5">yes</text>
+  </g>
+
+  <g transform="translate(800 1000)">
+    <use href="#decision" />
+    <text class="caption">
+      <tspan>Can add</tspan>
+      <tspan x="0" dy="1.2em">Remandment?</tspan>
+    </text>
+    <path class="arrow" d="M 0,30 v 50" />
+    <text x="5" y="50">no</text>
+    <path class="arrow" d="M 100,0 h 400" />
+    <text x="105" y="-5">yes</text>
+  </g>
+
+  <g transform="translate(1400 1000)">
+    <use href="#result" />
+    <text class="caption">Remandment</text>
+    <path class="arrow" d="M 100,0 h 100 v 300" />
+  </g>
+
+  <g transform="translate(800 1100)">
+    <use href="#jump" />
+    <text class="caption">H*</text>
+  </g>
+
+  <g transform="translate(800 1200)">
+    <use href="#decision" />
+    <text class="caption">
+      <tspan>Can add</tspan>
+      <tspan x="0" dy="1.2em">Reversion?</tspan>
+    </text>
+    <path class="arrow" d="M 0,30 v 50" />
+    <text x="5" y="50">no</text>
+    <path class="arrow" d="M 100,0 h 170" />
+    <text x="105" y="-5">yes</text>
+  </g>
+
+  <g transform="translate(1100 1200)">
+    <use href="#decision-xs" />
+    <text class="caption">DL</text>
+    <path class="arrow" d="M 0,30 v 150" />
+    <text x="-47" y="43">none</text>
+    <path class="arrow" d="M 30,0 h 170" />
+    <text x="35" y="-5">partial</text>
+    <path class="arrow" d="M 21,21 100,100 h 100" />
+    <text x="35" y="33">full</text>
+  </g>
+
+  <g transform="translate(1400 1200)">
+    <use href="#input" />
+    <text class="caption">
+      <tspan>Reversion reasons</tspan>
+      <tspan x="0" dy="1.2em" font-style="italic">RR</tspan>
+    </text>
+    <path class="arrow" d="M 0,20 v 60" />
+  </g>
+
+  <g transform="translate(1400 1300)">
+    <use href="#result" />
+    <text class="caption">Reversion</text>
+    <path class="arrow" d="M 100,0 h 100 v 300" />
+  </g>
+
+  <g transform="translate(800 1300)">
+    <use href="#jump" />
+    <text class="caption">H*</text>
+  </g>
+
+  <g transform="translate(1100 1400)">
+    <use href="#input" />
+    <text class="caption">
+      <tspan>Invalid Reversion</tspan>
+      <tspan x="0" dy="1.2em" font-style="italic">Help ticket</tspan>
+    </text>
+    <path class="arrow" d="M 100,0 h 100" />
+  </g>
+
+  <g transform="translate(1400 1400)">
+    <use href="#result" />
+    <text class="caption">Help</text>
+  </g>
+
+  <g transform="translate(200 1400)">
```

Comment:
Poradie boxov by malo byt zlava doprava a zhora dole. Result `Help` predbieha decision `Can add Dis / Ref / Adv / Ext?` Najde sa viac takych dvojic. Poradie boxov vsak ma svoju logiku, takze podla uvazenia to zmen alebo nechaj tak ako to je. Mozno by tiez bolo fajn posunut tento decision nizsie.

Reply:
@viliambalaz Ano, in general by jednotlive kroky mali byt v zdrojaku po riadkoch, nech nie je chaos (celkovo flow toho grafu sa snazi byt primarne zlava do prava a potom zhora dole). Ale konkretne v pripade `Can add Dis / Ref / Adv / Ext?` som tento stav posunul o riadok vyssie, nech neplytvame miestom, kedze bola tam velka volna plocha. Ale aj vizualne v grafe to vyzera "nizsie" ako je `Help` z predoslej vetvy. Cize aj v kode je lepsie ak `Can add Dis / Ref / Adv / Ext?` je pri stavoch, ktore s nim suvisia.